test/files/jvm/duration-tck.scala

```diff
@@ -174,6 +174,20 @@ object Test extends App {
   (1.second + 1.millisecond).unit mustBe MILLISECONDS
 
 
+  // test unit labels
+  for ((unit, labels) ← Seq(
+    DAYS         -> "d day",
+    HOURS        -> "h hour",
+    MINUTES      -> "m min minute",
+    SECONDS      -> "s sec second",
+    MILLISECONDS -> "ms milli millisecond",
+    MICROSECONDS -> "µs micro microsecond",
+    NANOSECONDS  -> "ns nano nanosecond"
```

Comment:
As we're adding tests let's add the plural forms as well.

Reply:
Done.